sdk-transaction/src/main/java/org/fisco/bcos/sdk/transaction/manager/AssembleTransactionProcessor.java

```diff
@@ -335,6 +338,35 @@ public String encodeFunction(String abi, String functionName, List<Object> param
         return abiCodec.encodeMethod(abi, functionName, params);
     }
 
+    @Override
+    public RawTransaction getDeployedRawTransaction(String abi, String bin, List<Object> params)
+            throws ABICodecException {
+        return transactionBuilder.createTransaction(
+                DefaultGasProvider.GAS_PRICE,
+                DefaultGasProvider.GAS_LIMIT,
+                null,
+                abiCodec.encodeConstructor(abi, bin, params),
+                BigInteger.ZERO,
+                new BigInteger(this.chainId),
+                BigInteger.valueOf(this.groupId),
+                "");
+    }
+
+    @Override
+    public RawTransaction getRawTransaction(
```

Comment:
Recommend to add BlockLimit as one of the params in case of the SDK is not connected with blockchain

Reply:
done